File: backend/app/Models/Note.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

class Note extends Model
{
    public $timestamps = false;

    protected $fillable = [
        "title",
        "text",
        "topic_id",
        "user_id",
        "role"
    ];

    public function topic(): BelongsTo
    {
        return $this->belongsTo(Topic::class);  
    }

    public function user(): BelongsTo
    {
        return $this->belongsTo(User::class);  
    }
}

File: tests/BolcsessegBarkaAPITests/ApiLibrary/Note.cs
using Newtonsoft.Json;

namespace ApiLibrary;

public class Note
{
    [JsonProperty("id")]
    public int Id { get; set; }
    
    [JsonProperty("title")]
    public string Title { get; set; }

    [JsonProperty("text")]
    public string Text { get; set; } 
    
    [JsonProperty("user_id")]
    public int UserId { get; set; } 
}